R/FilteredData.R

```diff
@@ -104,7 +134,7 @@ FilteredData <- R6::R6Class( # nolint
     #' @description
     #' Gets variable names of a given dataname for the filtering.
     #'
-    #' @param dataname (`character`) name of the dataset
+    #' @param dataname (`character(1)`) name of the dataset
     #' @return (`character` vector) of variable names
     get_filterable_varnames = function(dataname) {
```

Comment:
Btw, what is the point of this method? XD

Reply:
it's overwritten in CDISCFilteredData...

Reply:
- in FilteredData it doesn't do anything. colnames(data) == filterable_varnames
- In [CDISCFIltereData](https://github.com/insightsengineering/teal.slice/blob/8a09ad5aaeab460d921218cd4f80c9b0c1af1a1d/R/CDISCFilteredData.R#L157) we don't want child to have duplicated columns. It's actually users request because in ADTTE there are some duplicated columns comparing to ADSL. I foreseen that in the future it might cause some confusion as we can for example build a relationship between movie <-> actor and movie.name with actor.name are duplicated names but not the same thing. Let's discuss this in some time in the future if it will be important.